src/libs/actions/IOU.js

```diff
@@ -111,8 +111,34 @@ function createIOUSplit(params) {
         });
 }
 
+/**
+ * Settles an IOU Report and then retrieves the iou and chat reports to trigger updates to the UI.
+ */
+function settleIOUReport({
```

Comment:
Why is this called `settle...` when the API is called `pay...`?

Reply:
Not sure, we've been using both up to now. Changed every 'settle' reference to 'pay'.